development/extensions/skeleton_extension.rst

```diff
@@ -30,17 +30,17 @@ installed into a phpBB board just the same as any other.
 
 .. note::
 
-    The Skeleton Extension is a tool for extension developers. As such,
-    it is intended to be installed in a phpBB board used for
-    offline/local development of extensions. It is not necessarily
-    intended to be installed on live/production web sites.
+    The Skeleton Extension is intended to be installed in a phpBB
+    board used for offline/local development of extensions. It is
+    not necessarily intended to be installed on live/production web
```

Comment:
First, bots are not authorized to view the page. Second, who cares if they did? For crawling, it's a non-issue. For spam, it's no more an issue than when they post. So that goes back to the original point, this isn't intended for live sites, but those who know what they're doing and want it on alive site for good reason can totally do so.